src/Spreadsheet.js

```diff
@@ -109,11 +110,10 @@ class Spreadsheet<CellType, Value> extends PureComponent<{|
     DataViewer,
     DataEditor,
     getValue,
-    getBindingsForCell
+    getBindingsForCell,
+    formulaParser: new FormulaParser()
```

Comment:
This is problematic because it will be shared between the instances. Can you do this on init instead?